site/constants/index.ts

```diff
@@ -31,10 +33,17 @@ export const DISRUPTION_REASONS: {
     },
 ];
 
+export const contactFeedbackQuestion = "Did you contact us for assistance at any point?";
+export const solveFeedbackQuestion = "Did we solve your problem?";
+export const hearAboutUsFeedbackQuestion = "How did you hear about our service?";
+export const generalFeedbackQuestion = "Please let us know any feedback or suggestions for improvements you may have";
+
 export const COOKIES_POLICY_COOKIE = "cdd-cookies-policy";
 
 export const COOKIE_PREFERENCES_COOKIE = "cdd-cookie-preferences-set";
 
 export const oneYearInSeconds = 31556952;
```

Comment:
Might be worth updating this to capitals as well

Reply:
done